homeworks/lesson2/app/src/components/course/list-course/list-course.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {ListGroup, ListGroupItem} from 'react-bootstrap';
+import './list-course.css';
+
+class Courses extends React.Component {
+
+    constructor(props, context) {
+        super(props, context);
+        this.state = {
+            courses: [
+                {
+                    id: 1,
+                    title: 'Course 1',
+                    desc: 'Intro'
+                },
+                {
+                    id: 2,
+                    title: '2nd Course',
+                    desc: 'basic knowledge'
+                },
+                {
+                    id: 3,
+                    title: 'The Last course',
+                    desc: 'Jedi knowledge'
+                }
+            ]
+        };
+    }
+
+    render() {
+        const {
+            courses
+        } = this.state;
+
+        return (
+            <div className="courses">
+                <h1>Courses</h1>
+                <ListGroup>
+                    {this.renderCourses(courses)}
+                </ListGroup>
+            </div>
+        );
+    }
+
+    renderCourses(coursesList) {
+        return coursesList.map(course => {
+            return <ListGroupItem
+                href={'/courses/' + course.id}
+                header={course.title}
+                key={course.id + '-key'}
```

Comment:
Why are you add string '-key' to ID which should be unique and is not necessarily add to it extra data?

Reply:
![image](https://user-images.githubusercontent.com/15190666/46913575-9feeb500-cf98-11e8-8d91-a242a7db23fe.png)